src/iptax/ai/review.py

```
@@ -0,0 +1,477 @@
+
+COLORS = {
+    Decision.INCLUDE: "green",
+    Decision.EXCLUDE: "red",
+    Decision.UNCERTAIN: "yellow",
+}
+
+
+class ReviewResult:
+    """Result of the review process."""
+
+    def __init__(self, judgments: list[Judgment], accepted: bool = False) -> None:
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)